constellation_base/models.py

```diff
@@ -39,3 +42,20 @@ def settings_dict(self):
             'organization': self.organization,
             'titlebar_links': self.titlebar_links,
         }
+
+
+class LocalGroupACL(models.Model):
+    """ACL for Local Groups that don't exist in other authentication sources"""
+    group = models.ForeignKey(Group)
```

Comment:
Why are we tracking per users per group  instead of groups per user? ie keep track of all the groups that a user is in, instead of all the users in each group...Doesn't matter too much